wip (#79204)
vercel/next.js: .changeset/shaggy-owls-visit.md

---
'next': patch
---

> Fixed rewrite params of the interception routes not being parsed correctly when deployed to Vercel

Reviewer: I think we can generalize this a bit

```suggestion
Fixed rewrite params of the interception routes not being parsed correctly in certain deployed environments
```
